m365/powerbi/README.md

```diff
@@ -0,0 +1,116 @@
+## **SCuBA PowerBI Integration Steps**
+
```

Comment:
Could use a brief intro on what this is for and why someone would want to do it. A screenshot of the report would help.